src/compiler/emitter.ts

```diff
@@ -3,6 +3,190 @@
 
 /* @internal */
 namespace ts {
+    export const defaultInitCompilerOptions: CompilerOptions = {
+        module: ModuleKind.CommonJS,
+        target: ScriptTarget.ES3,
+        noImplicitAny: true,
+        outDir: "built",
+        rootDir: ".",
+        sourceMap: false,
+    }
+
+    export function buildConfigFile(writer: EmitTextWriter, compilerOptions: CompilerOptions, fileNames?: string[], excludes?: string[]) {
+        compilerOptions = extend(compilerOptions, defaultInitCompilerOptions);
+        let { write, writeLine, increaseIndent, decreaseIndent } = writer;
+        let { optionNameMap } = getOptionNameMap();
+        writeTsConfigDotJsonFile();
+
+        function writeTsConfigDotJsonFile() {

```

Comment:
can we split this change into two, one that does the `--init` and one that does the comments in json